lldb/include/lldb/DataFormatters/FormatCache.h

```diff
@@ -45,15 +45,12 @@ class FormatCache {
     void Set(lldb::TypeSummaryImplSP);
     void Set(lldb::SyntheticChildrenSP);
   };
-  typedef std::map<ConstString, Entry> CacheMap;
-  CacheMap m_map;
+  std::map<ConstString, Entry> m_entries;
```

Comment:
Could this be a StringMap, or if we benefit from ConstString here, a DenseMap<ConstString, Entry> for faster lookups?